api/nginx/nginx.conf

```diff
@@ -8,88 +11,13 @@ events {
 }
 
 http {
-  upstream managed-grpc {
```

Comment:
Where it was moved?

Reply:
See the added comment on top of the file